README.md

```diff
@@ -1,12 +1,71 @@
 # pred_missing_celltypes
-Predicting missing cell-types from deconvolution reference using residual. 
+### Predicting missing cell-types from deconvolution reference using residual. 
 
-This repository is currently a work in progress of a project aimed at uncovering information from missing cell types, after RNA-bulks have been deconvoluted. 
-Currently, NNLS has been used for deconvolution, and the residual is being analyzed as three different matrices (details in exploratory_experiments). The residual is being factoried/reduced using SVD, PCA, ICA and NMF. 
+This repository is currently a work in progress of a project aimed at uncovering information from missing cell types, after RNA-bulks have been deconvoluted. Our study aims to answer the question of what happens when we have real and accumulated cells missing from the single-cell dataset in bulk deconvolution methodologies, and whether we are able to retrieve the missing cell-type’s proportions through the calculated proportions. 
 
-Folders:
-preprocessing/ : notebook to process data and create pseudobulks for experiments.
-exploratory_experiments/ :  notebooks containing the deconvolution and factorization of different types of pseudobulks.
-functions/ :  scripts with functions used throughout.
 
+## Folder Information:
+- /preprocessing/ : notebook to process data and create pseudobulks for experiments.
```

Comment:
you can make paths look like code by surrounding them in backticks

Reply:
nice i didn't know, added backticks to all paths.